rclpy/rclpy/__init__.py

```diff
@@ -214,6 +217,7 @@ def create_node(
     namespace: Optional[str] = None,
     use_global_arguments: bool = True,
     enable_rosout: bool = True,
+    rosout_qos_profile: Optional[Union[QoSProfile, int]] = qos_profile_system_default,
```

Comment:
I don't think this is the correct "default" profile for the rosout publisher.  As far as I can tell, the default here is actually https://github.com/ros2/rcl/blob/d9daca7c0af7ecdcd103ece30bb7c64bc919ef24/rcl/include/rcl/logging_rosout.h#L37-L48 .  So I think we should define this to be the same by default.

Reply:
addressed in https://github.com/ros2/rclpy/pull/1376/commits/987bc366b2c2173925a95f4bed342cf91f25b9f7